src/common/hooks.js

```diff
@@ -78,7 +78,11 @@ export function useLocalStorage(key, initialValue) {
       setStoredValue(valueToStore);
       // Save to local storage
       if (!isServer) {
-        window.localStorage.setItem(key, JSON.stringify(valueToStore));
+        if (valueToStore === null) {
```

Comment:
Should this be `valueToStore === undefined`? Why not save `null` values?

Reply:
Yeah im open to either one... or we could also pass back an array of 3 values with an explicit clear function so something like

```
const [token, setToken, clearToken] = useLocalStorage('token', false)
```